tox.ini

```diff
@@ -13,6 +13,7 @@ env_list =
     coverage
     readme
     docs
+    3.13t
```

Comment:
Would we not need support in tox for this?

Reply:
I think I found a way by using an env var that points to the python executable and using that to explicitly configure tox with `base_python` to use that Python installation. I'm working on it.

Reply:
@gaborbernat maybe I've got something working now. Could you please re-trigger the CI?